easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.2-foss-2015b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.1.2'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
+
+description = """Xerces-C++ is a validating XML parser written in a portable
+subset of C++. Xerces-C++ makes it easy to give your application the ability to
+read and write XML data. A shared library is provided for parsing, generating,
+manipulating, and validating XML documents using the DOM, SAX, and SAX2
+APIs."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['xerces-c-%(version)s.tar.gz']
+source_urls = [
+    'http://archive.apache.org/dist/xerces/c/%(version_major)s/sources/'
+]
+
+dependencies = [
+    ('cURL', '7.44.0'),
```

Comment:
Changed it to use 7.45.0, because 7.47.1 isn't available yet in my local copy...